docs/wiki/modules/tenancy.md

```
@@ -0,0 +1,26 @@
+All paths relative to `backend/app/`.
+
+## Routes
+- `api/routes/login.py`, `api/routes/auth.py` — login/JWT
+- `api/routes/users.py`, `api/routes/user_project.py` — users + membership
+- `api/routes/organization.py`, `api/routes/project.py`
+- `api/routes/api_keys.py`
+- `api/routes/onboarding.py`
+- `api/routes/private.py` — internal endpoints
+
+## Tables (SQLModel)
+| Table | Model |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)